runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculator.java

```diff
@@ -89,6 +89,7 @@
 		searchBar.setBackground(ColorScheme.DARKER_GRAY_COLOR);
 		searchBar.setHoverBackgroundColor(ColorScheme.DARK_GRAY_HOVER_COLOR);
 		searchBar.addKeyListener(e -> onSearch());
```

Comment:
Is there any problem with just replacing this line with:
```
searchBar.getDocument().addDocumentListener(new DocumentListener()
{
	@Override
	public void insertUpdate(DocumentEvent e)
	{
		onSearch();
	}

	@Override
	public void removeUpdate(DocumentEvent e)
	{
		onSearch();
	}

	@Override
	public void changedUpdate(DocumentEvent e)
	{
		onSearch();
	}
});
```
Or would that allow us to skip the addition of the mouse listeners?

Reply:
Didn't think of that. That works too since clearing the textfield would mean changing the document. I'll make those changes?